cli/src/semgrep/commands/scan.py

```diff
@@ -606,19 +606,31 @@ def scan_options(func: Callable) -> Callable:
     # help="contact support@r2c.dev for more information on this"
 )
 @click.option(
-    "--pro",
+    "--interproc",
     is_flag=True,
     hidden=True
     # help="contact support@r2c.dev for more information on this"
 )
 @click.option(
-    "--interproc",
+    "--interfile",
     is_flag=True,
     hidden=True
     # help="contact support@r2c.dev for more information on this"
 )
 @click.option(
-    "--interfile",
+    "--pro-languages",
+    is_flag=True,
+    hidden=True
+    # help="contact support@r2c.dev for more information on this"
+)
+@click.option(
+    "--pro-intrafile",
```

Comment:
What is the difference between this and `--interproc`?

Reply:
No difference, just synonyms. I'm a bit confused by so many flag renaming, I'm not sure whether to keep it around or deprecate `--interproc` and `--interfile` like we did with `--deep`.

Reply:
We should have exactly one non-deprecated flag for each behavior. We can keep the old ones around for a while, but we should clearly mark them as deprecated in the help and point people towards the preferred flag.

Reply:
OK added help messages for the three `--pro*` ones and deprecated the others.